plugins/cue_points/base/lib/model/CuePoint.php

```diff
@@ -437,4 +437,19 @@ public function hasPermissionToCopyToEntry( entry $entry )
 
 		return false;
 	}
+
+	/**
+	 * @param      string $name
+	 * @param      string $namespace
+	 * @return     boolean True if $name has been modified.
+	 */
+	public function isCustomDataModified($name = null, $namespace = '')
```

Comment:
The object is not reliable, the oldCustomDataValues could be changed in first consumer and later be checked by a different consumer.